chrome-finder.ts

```diff
@@ -155,8 +155,11 @@ export function win32() {
   const suffixes = [
     '\\Google\\Chrome SxS\\Application\\chrome.exe', '\\Google\\Chrome\\Application\\chrome.exe'
   ];
-  const prefixes =
-      [process.env.LOCALAPPDATA, process.env.PROGRAMFILES, process.env['PROGRAMFILES(X86)']];
+  const prefixes = [process.env.LOCALAPPDATA, process.env.PROGRAMFILES];
```

Comment:
rather than this why not just do this at the end of the array:

```js
.filter(Boolean)
```

Helps in case we need to add one more env variable in the future?

Reply:
That's certainly a more general solution. Will change.